core/object.h

```diff
@@ -145,6 +145,9 @@ struct PropertyInfo {
 	PropertyHint hint = PROPERTY_HINT_NONE;
 	String hint_string;
 	uint32_t usage = PROPERTY_USAGE_DEFAULT;
+#ifdef TOOLS_ENABLED
+	String doc_string;
+#endif
```

Comment:
@reduz Please review this `object.h` addition.

Reply:
Please do not add more stuff to these structures, use a different approach to obtain documentation from scripts.

Reply:
done. descriptions no more depend on it's propertyinfo/methodinfo. (they directly changed to `DocData::*Doc` at compile)